src/PlaywrightSharp/Chromium/IChromiumBrowser.cs

```diff
@@ -34,56 +34,57 @@ public interface IChromiumBrowser : IBrowser
         /// <returns>A <see cref="Task"/> that completes when the browser session was created, yielding the new session.</returns>
         Task<ICDPSession> NewBrowserCDPSessionAsync();
 
-        /// <inheritdoc cref="IBrowser.NewContextAsync(ViewportSize, string, bool?, bool?, string, Geolocation, ContextPermission[], bool?, bool?, decimal?, Credentials, bool?, bool?, bool?, ColorScheme?, string, Dictionary{string, string}, RecordHarOptions, RecordVideoOptions, ProxySettings, string, StorageState)"/>
```

Comment:
@kblok I could use your opinion on this one especially. It feels like this isn't needed anymore, but maybe I'm missing something?

Reply:
@yury-s are we dropping support for `backgroundPage` and `serviceWorker`  in bindings?
If so we can drop this interface and close #1235 and #1234
`

Reply:
Happy to drop this, if we can, but I'll leave it to a separate PR.